omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -230,36 +238,41 @@
   "nginx",
   "keepalived"
 ].each do |service|
-  if node["private_chef"][service]["enable"]
-    include_recipe "private-chef::#{service}"
+  if node["private_chef"][service]["external"]
+    begin
+      # Perform any necessary configuration of the external service:
+      include_recipe "private-chef::#{service}-external"
```

Comment:
The flip side is that wanting to avoid the  'external' conditionals throughout the cookbooks if we  can help it - treating things the same as much as possible makes it easier to wrap your head around it.  Any ideas for  a different approach that still lets this happen?